monai/run_inference_single_image.py

```diff
@@ -318,7 +354,7 @@ def main():
 
```

Comment:
TODO: add function to keep only the largest connected component after averaging the flipped predictions

Reply:
Done in commit https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/96/commits/71d34aaae336e2b522205e07b7ab181a9ebeacd2